elasticsearch/templates/_helpers.tpl

```diff
@@ -0,0 +1,16 @@
+{{/* vim: set filetype=mustache: */}}
```

Comment:
Yes will delete this file if no usage found. Keeping it right now just in case if needed.